tests/dialects/test_bigquery.py

```diff
@@ -1844,12 +1839,6 @@ def test_user_defined_functions(self):
             "CREATE TEMPORARY FUNCTION string_length_0(strings ARRAY<STRING>) RETURNS FLOAT64 LANGUAGE js OPTIONS (library=['gs://ibis-testing-libraries/lodash.min.js']) AS '\\'use strict\\'; function string_length(strings) { return _.sum(_.map(strings, ((x) => x.length))); } return string_length(strings);'",
         )
 
-    def test_group_concat(self):
-        self.validate_all(
-            "SELECT a, GROUP_CONCAT(b) FROM table GROUP BY a",
-            write={"bigquery": "SELECT a, STRING_AGG(b) FROM table GROUP BY a"},
-        )
-
```

Comment:
Why was this removed? I think it was [intentionally added](https://github.com/tobymao/sqlglot/pull/862) to test legacy BigQuery SQL.